ffi/Cargo.toml

```diff
@@ -15,6 +15,9 @@ crate-type = [ "cdylib", "rlib" ]
 doctest = false
 bench = false
 
+[build-dependencies]
```

Comment:
Sure, I don't have a strong opinion, I just ordered them in additive order since build-dependencies are required by dependencies which in turn are required by dev-dependencies